### ISSUE TITLE
crs_faq outline

### ISSUE THREAD

**Issue by EconomiCurtis** [batpigandme/crs_faq#1]
Seems like we need three categories, guidelines, FAQs, and a few canned responses.

**Comment by batpigandme** [comment]
Makes sense. We can totally port this over to an rstudio repo, too— I just thought it'd make more sense for you to admin that one! 👍
